sslr-examples/src/main/java/org/sonar/sslr/examples/grammars/typed/JsonPunctuator.java

```diff
@@ -0,0 +1,28 @@
+package org.sonar.sslr.examples.grammars.typed;
+
+import org.sonar.sslr.grammar.GrammarRuleKey;
+
+enum JsonPunctuator implements GrammarRuleKey {
```

Comment:
this class doesn't bring value on its own.

I'd simply drop it and move the `LCURLYBRACE`, `COMMA`, etc. declarations to `JsonLexer`